crates/sui-core/src/tbls/mod.rs

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
What does tbls mean? would you be able to add some doc comment explaining what it is?

Reply:
☝️

Reply:
added some info in the mod file (tbls_ids.rs has more information)